src/main/resources/templates/recipe/create.html

```diff
@@ -54,6 +54,13 @@
 				<label th:text="${category.name}"></label>
 			</th:block>
 		</div>
+		<div class="form-group">
+			<label th:for="beverages">Sommelier Suggestions</label>
```

Comment:
Minor point, but maybe beverage pairings since we're not dealing exclusively in wine?

Reply:
In modern times, a sommelier's role may be considered broader than working only with wines, and may encompass all aspects of the restaurant's service, with an enhanced focus on wines, beers, spirits, soft-drinks, cocktails, mineral waters, and tobaccos.  (Wikipedia isn't the only source, just the most succinct!)  If you find it confusing, I can rename as beverage pairings rename.

Reply:
Dig it! Learned something new today. No need to change on my account!